src/com/theironyard/novauc/Main.java

```diff
@@ -1,8 +1,150 @@
+        ps.setString(1,bestDish);
+        ps.setInt(2,restNumWaiters);
+        ps.execute();
+    }
+
+    public static User currentUser() {
+        return new User(1, "billyray");
+    }
+
+    public static HashMap<String, User> accountInfo = new HashMap<>();
+    public static ArrayList<Restaurant> restAL = new ArrayList<>();
+
+    public static void main(String[] args) throws SQLException {
+
+        Spark.init();
+        Server.createWebServer().start();
+        createTables();
+
+        Spark.get("/", ((request, response) -> {
+            Session session = request.session();
+            String name = session.attribute("userName");
+            HashMap userActivity = new HashMap();
+            if(!accountInfo.containsKey(name)) {
+                return new ModelAndView(userActivity,"index.html");
+                }
+            else {
+                  userActivity.put("entries", selectRestaurants());
+                  userActivity.put("userName", name);
+                  return new ModelAndView(userActivity, "index.html");
+                  }
+                }),
+                new MustacheTemplateEngine()
+        );
+
+        Spark.post("/login", (request, response) -> {
+            String name = request.queryParams("userName");
+            String password = request.queryParams("passwordLogin");
+            Session session = request.session();
+
+            if(accountInfo.containsKey(name)) {
+                if(password.equals(accountInfo.get(name).getPassword())) {
+                    session.attribute("userName", name);
+                }
+            }
+            else {
+                session.attribute("userName", name);
+                accountInfo.put(name, new User(name, password));
+            }
+            response.redirect("/");
+            return "";
+        });
+
+        Spark.post("/create-restaurant", (request, response) -> {
+
+            String restName = request.queryParams("restName");
+            String bestDish = request.queryParams("bestDish");
+            int restNumWaiters = Integer.valueOf(request.queryParams("restNumWaiters"));
+
+            insertRestaurant(restName, bestDish,restNumWaiters);
+
+            response.redirect("/");
+            return "";
+        });
+
+        Spark.post("/edit-restaurant", (request, response) -> {
+
+            //int id = Integer.valueOf(request.queryParams("restEditId"));
+            String restName = request.queryParams("restEditName");
+            String bestDish = request.queryParams("restEditDish");
+            int restNumWaiters = Integer.valueOf(request.queryParams("restEditWaiters"));
+
+            updateRestaurant(restName, bestDish, restNumWaiters);
+
+            response.redirect("/");
+            return "";
+        });
+
+        Spark.post("/delete-restaurant", (request, response) -> {
+            deleteRestaurant(Integer.valueOf(request.queryParams("restDelete")));
+            response.redirect("/");
```

Comment:
Clean and concise.  Nice use of all your knowledge.  I like the mustache use so that you only need one html page.  Awesome job.